MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Hyperlink.xaml

```diff
@@ -59,4 +61,76 @@
         <Setter Property="FontSize" Value="112"/>
         <Setter Property="FontWeight" Value="Light"/>
     </Style>
+    
+    <!-- updated specification 2018 -->
+    <Style TargetType="{x:Type Hyperlink}" x:Key="MaterialDesign2Body1Hyperlink">
+        <Setter Property="FontSize" Value="16" />
+        <Setter Property="FontWeight" Value="Regular" />
+        <Setter Property="TextDecorations" Value="None" />
+        <Setter Property="Foreground" Value="{DynamicResource MaterialDesignBody}" />
+        <Style.Triggers>
+            <Trigger Property="IsEnabled" Value="False">
+                <Setter Property="Foreground" Value="{DynamicResource MaterialDesignCheckBoxDisabled}" />
+            </Trigger>
+            <Trigger Property="IsEnabled" Value="True">
+                <Setter Property="Cursor" Value="Hand" />
+            </Trigger>
+            <MultiTrigger>
+                <MultiTrigger.Conditions>
+                    <Condition Property="IsEnabled" Value="True" />
+                    <Condition Property="IsMouseOver" Value="True" />
+                </MultiTrigger.Conditions>
+                <Setter Property="TextDecorations" Value="Underline" />
+            </MultiTrigger>
+        </Style.Triggers>
+    </Style>
+    <Style TargetType="{x:Type Hyperlink}" x:Key="MaterialDesign2Body2Hyperlink"
+           BasedOn="{StaticResource MaterialDesignBody1Hyperlink}">
```

Comment:
It looks like several of these styles are still being based off of the old hyperlink style. Is that intentional?